src/app/events/components/test-edit-grades/test-edit-grades-common.scss

```diff
@@ -47,10 +58,33 @@ td {
     display: none;
   }
 
+  table,
+  tbody,
+  thead {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+  }
+
+  th,
+  tr {
+    display: flex;
+    width: 100%;
+  }
+
   .student-name {
-    // Make sure the student name is going to ellipse also on mobile
-    max-width: var(--student-name-column-width);
+    flex: 1;
+    display: block;
   }
+
+  .test-grade {
+    flex: none;
+    display: block;
+  }
+
+  //tr:not(.header-collapsible) th.test-grade  {
+  //  width: calc($test-grade-title-column-width + 4em);
+  //}
```

Comment:
Hast du das bewusst drin gelassen?

Reply:
Nein, das habe ich in der Eile übersehen - danke 🙏